guardian/editorial-production-metrics
# Upgrade to Play 2.7

## What does this change?

- Upgrade to Play 2.7
- Upgrade to latest version of `panda-hmac`
- Replace `configuration-magic` with `simple-configuration`

## Background
The `panda-hmac` library must be bumped to resolve a vulnerability, but this library no longer supports Play 2.6, so we need upgrade to Play 2.7. 

Upgrading Play has a knock-on effect in that the `magic-configuration` library only supports Play 2.6, so we are migrating to `simple-configuration` (as recommended by DevX). 

The `simple-configuration` library does not support storing config in DynamoDB, so this PR is accompanied by moving our private app configuration into AWS Parameter Store.

## How to test

Currently deployed to CODE. The graphs and API responses have been checked to ensure everything in the frontend still works. Auth has been tested and Kibana logs checked to see there are no errors generated by the upgrades.

## How can we measure success?

There should be no change or interruption caused by this PR.
